Lib/idlelib/colorizer.py

```diff
@@ -291,6 +307,51 @@ def recolorize_main(self):
                     if DEBUG: print("colorizing stopped")
                     return
```

Comment:
Coverage says the conditional above is execution both ways, the the return is not.  This is the only thing short of 100%. I am going to assume that this is a glitch in my older version of coverage.